src/Abp.Web/Auditing/WebClientInfoProvider.cs

```diff
@@ -102,9 +102,23 @@ protected virtual string GetComputerName()
             }
         }
 
-        public virtual HttpContextBase GetCurrentHttpContext()
+        public virtual HttpRequestBase GetCurrentHttpRequest()
         {
-            return HttpContext.Current == null ? null : new HttpContextWrapper(HttpContext.Current);
+            try
+            {
+                var httpContext = HttpContext.Current == null ? null : new HttpContextWrapper(HttpContext.Current);
```

Comment:
Should leave this line outside since it's not expected to throw exception.

Reply:
👍